src/Homework4/HousePlant.java

```diff
@@ -0,0 +1,55 @@
+package Homework4;
+
+public class HousePlant extends Plant {
+    private String housePlantName;
```

Comment:
These fields looks similar to me in subclasses. I'd rather move them to superclass

Reply:
Moved fields to Plant.